setup.cfg

```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = flaskr
+version = 1.0.0
+url = https://flask.palletsprojects.com/tutorial/
+license = BSD-3-Clause
+maintainer = Pallets
+maintainer_email = contact@palletsprojects.com
+description = The basic blog app built in the Flask tutorial.
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+
+[options]
+packages = find:
+include_package_data = true
+install_requires =
+    Flask
+
+[options.extras_require]
```

Comment:
Since there aren't any tests, can this block and the following two others be removed?